File: rust/examples/sets/union.rs
use std::collections::HashSet;

fn main() {
    let mut english: HashSet<String> = HashSet::new();
    let mut spanish: HashSet<String> = HashSet::new();

    for word in ["door", "car", "lunar", "era"] {
        english.insert(word.to_string());
    }
    for word in ["era", "lunar", "hola"] {
        spanish.insert(word.to_string());
    }

    println!("{:?}", &english);
    println!("{:?}", &spanish);

    println!("{:?}", english.union(&spanish));
}

File: python-programming/src/examples/sets/union.py
english = set(['door', 'car', 'lunar', 'era'])
spanish = set(['era', 'lunar', 'hola'])

all_the_words = english.union(spanish)

print(english)
print(spanish)
print(all_the_words)

# x = english + spanish # TypeError: unsupported operand type(s) for +: 'set' and 'set'
